src/db/models/user.model.js

```diff
@@ -172,5 +264,40 @@ module.exports = (sequelize, DataTypes) => {
             });
     };
 
+    User.prototype.updateVerifyAccountToken = function updateVerifyAccountToken(email, expTime) {
```

Comment:
Why did you break apart the functionality found here in `User.prototype.updateVerifyAccountToken`, out of  `User.verifyAccountToken`? I would think it would make sense to have it all in `User.verifyAccountToken`. I say this because it's the same logical piece of functionality, and `updateVerifyAccountToken` is only called from `verifyAccountToken`.

Reply:
Resolved. See my comment today, 34 minutes ago. about the resolution of problem 2.A below (which is this same problem).